go.mod

```diff
@@ -9,31 +9,26 @@ require (
 	github.com/onsi/gomega v1.10.1
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
-	github.com/tsenart/vegeta/v12 v12.8.4
-	k8s.io/api v0.24.1
-	k8s.io/apimachinery v0.24.1
-	k8s.io/client-go v0.24.1
+	github.com/tsenart/vegeta/v12 v12.8.3
+	k8s.io/api v0.22.17
+	k8s.io/apimachinery v0.22.17
+	k8s.io/client-go v0.22.17
```

Comment:
Why are we downgrading?

Reply:
bc this off GED-kubedog-estela branch, where we [changed](https://github.com/keikoproj/kubedog/commit/2b3d092c8f1dc2e66f5f72b677ae64255664344e) the dependencies to match arktika. These shouldn't be part of the PR.